aldryn_newsblog/models.py

```diff
@@ -137,7 +149,7 @@ class Article(TranslatedAutoSlugifyMixin,
     tags = TaggableManager(blank=True)
 
     related = SortedManyToManyField('self', verbose_name=_('related articles'),
-                                    blank=True)
+                                    blank=True, symmetrical=ARTICLE_RELATED_SYMMETRICAL)
```

Comment:
The users themselves will not be able to add a migration to addons. It goes into the package directory under /var/lib/python and not under /app and it will be lost with each docker container run. Or am I missing anything?